core/renderers/geras/path_object.js

```diff
@@ -66,7 +67,40 @@ Blockly.geras.PathObject = function(root) {
    */
   this.svgPathLight = Blockly.utils.dom.createSvgElement('path',
       {'class': 'blocklyPathLight'}, this.svgRoot);
+
+
+  /**
+   * Primary colour of the block in '#RRGGBB' format.
+   * @type {string}
+   * @package
+   */
+  this.primaryColour = '#000000';
+
+  /**
+   * Secondary colour of the block in '#RRGGBB' format.
+   * Used for the body of a shadow block in Geras.
+   * @type {string}
+   * @package
+   */
+  this.secondaryColour = '#000000';
+
+  /**
+   * Tertiary colour of the block in '#RRGGBB' format.
+   * Used for the light path (highlight) in Geras.
+   * @type {string}
+   * @package
+   */
+  this.tertiaryColour = '#000000';
+
+  /**
+   * The colour of the dark path on the block in '#RRGGBB' format.
+   * @type {string}
+   * @package
+   */
+  this.darkColour = '#000000';
 };
+Blockly.utils.object.inherits(Blockly.geras.PathObject,
```

Comment:
Add require for ``Blockly.utils.object``

Reply:
Done.